sdk/personalizer/ai-personalizer-rest/test/public/aadCredentialTests.spec.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Recorder, env } from "@azure-tools/test-recorder";
+import { createRecorder } from "./utils/recordedClient";
+import { Context } from "mocha";
+import { createTestCredential } from "@azure-tools/test-credential";
+import createPersonalizerClient, { PersonalizerClient, isUnexpected } from "../../src";
+
+describe("AAD Tests", () => {
+  let recorder: Recorder;
+  let client: PersonalizerClient;
+
+  beforeEach(async function (this: Context) {
+    recorder = await createRecorder(this);
+    client = createPersonalizerClient(
+      env["PERSONALIZER_ENDPOINT_SINGLE_SLOT"] ?? "",
+      createTestCredential(),
+      recorder.configureClientOptions({})
+    );
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  // test is failing browser mode. Needs investigation.
+  it.skip("get configuration using aad authentication test", async function () {
```

Comment:
This test fails in browser mode but succeeds in node however. I used AD app + secret to test this.

Reply:
What errors are you getting?

Reply:
Fails only in the browser?

Reply:
Actually, all the browser tests are failing now. Sorry for misleading. Following up offline with Jose.

Reply:
Fixed! Thanks to Harsha.